db/migrations/20200203183040_initial.js

```diff
@@ -28,10 +28,10 @@ exports.up = function(knex) {
       table.integer('color_2_id').unsigned();
```

Comment:
I'm still skeptical if this is the best way to go about this. Your thoughts on having the projects holding color IDs, or just storing the hex code correctly? We can discuss this more in detail in person, but I'm not sure I'm married to this initial setup of mine...